hack/build/build.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
not sure about this change.. in any other kubevirt project we use bash, not sh. Why is this required?

Reply:
You’re correct. I must have left it that way while testing my changes, though I can’t recall the exact reasons. I tested it with bash, and it works. The change has been reverted.